torchaudio/models/wavernn.py

```diff
@@ -1,18 +1,27 @@
-from typing import List, Tuple
+from typing import List, Tuple, Any
 
 import torch
 from torch import Tensor
 from torch import nn
 
+from ._utils import load_state_dict_from_url
+
+
 __all__ = [
     "ResBlock",
     "MelResNet",
     "Stretch2d",
     "UpsampleNetwork",
     "WaveRNN",
+    "wavernn",
 ]
 
 
+model_urls = {
+    'wavernn': 'https://download.pytorch.org/models/audio/wavernn_10k_epochs_8bits_ljspeech.pth',
+}
```

Comment:
in case this line is too long:
```
model_urls = {
    'wavernn_10k_epochs_8bits_ljspeech': (
           'https://download.pytorch.org/models/audio/'
           'wavernn_10k_epochs_8bits_ljspeech.pth'
     ),
}
```